alembic/versions/9371b45451b_remove_explosiveness_and_suspicious.py

```diff
@@ -0,0 +1,61 @@
+"""Remove everything that has to do with explosive and suspicious crashes.
+
+Revision ID: 9371b45451b
+Revises: 2990eb43b269
+Create Date: 2016-01-20 13:25:27.414375
+
+"""
+
+from alembic import op
+from socorro.lib.migrations import load_stored_proc
+
+# revision identifiers, used by Alembic.
+revision = '9371b45451b'
+down_revision = '2990eb43b269'
+
+
+def upgrade():
+    op.execute("""
+        DROP TABLE suspicious_crash_signatures
+    """)
+    op.execute("""
+        DROP FUNCTION backfill_explosiveness
+    """)
+    op.execute("""
+        DROP FUNCTION update_explosiveness
+    """)
+    op.execute('COMMIT')
+    load_stored_proc(op, ['backfill_matviews.sql'])
+
+
+def downgrade():
+    op.execute("""
+        CREATE TABLE suspicious_crash_signatures (
+            suspicious_crash_signature_id integer NOT NULL,
+            signature_id integer,
+            report_date timestamp with time zone
+        )
+    """)
+    # This doesn't actually have to be the real function. Just something so the
+    # downgrade works in alembic integration testing in Travis.
+    op.execute("""
+        CREATE OR REPLACE FUNCTION backfill_explosiveness(updateday date)
+        RETURNS boolean
+        LANGUAGE plpgsql
+            AS $$
+        BEGIN
+        RETURN TRUE;
+        END; $$;
+    """)
+    op.execute("""
+        CREATE OR REPLACE FUNCTION update_explosiveness(updateday date)
+        RETURNS boolean
+        LANGUAGE plpgsql
+            AS $$
+        BEGIN
+        RETURN TRUE;
+        END; $$;
+    """)
```

Comment:
Why don't you just run the files containing the actual functions? I don't think the idea of this is simply to test, but to keep some sort of way to go back in time. It might not be useful to us, but maybe someone outside Mozilla will be relying on this?